th-TH.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+"SpringBoard will be restarted." = "SpringBoard จะเริ่มต้นใหม่."
+"Unable to get task port for backboardd." = "ไม่สามารถรับค่า task port สำหรับพื้นหลัง."
+"Unable to make backboardd exit." = "ไม่สามารถสร้าง backboardd ออก."
+"Cleaned diagnostics data." = "ล้างข้อมูลการวินิจฉัย."
+"Preference was changed. The app will now exit." = "การตั้งค่าถูกเปลี่ยน. ออกจากแอพ."
+"Preferences were reset. The app will now exit." = "การตั้งค่าถูกรีเซ็ต. ออกจากแอพ."
+"Load Tweaks" = "เรียกใช้งาน Tweaks"
+"Description:\n\nThis option makes Substrate load extensions that are commonly referred to as tweaks in newly started processes.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64 SoCs (A7-A11)." = "Description:\n\nThis option makes Substrate load extensions that are commonly referred to as tweaks in newly started processes.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64 SoCs (A7-A11)."
+"Load Daemons" = "เรียกใช้งาน Daemons"
+"Description:\n\nThis option makes the jailbreak load the launch daemons located at /Library/LaunchDaemons and execute files located at /etc/rc.d.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64/arm64e SoCs (A7-A12X)." = "Description:\n\nThis option makes the jailbreak load the launch daemons located at /Library/LaunchDaemons and execute files located at /etc/rc.d.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64/arm64e SoCs (A7-A12X)."
+"Dump APTicket" = "เก็บค่า APTicket"
+"Description:\n\nThis option makes the jailbreak create a copy of the system APTicket located at /System/Library/Caches/apticket.der at its Documents directory which is accessible via iTunes File Sharing.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64/arm64e SoCs (A7-A12X)." = "Description:\n\nThis option makes the jailbreak create a copy of the system APTicket located at /System/Library/Caches/apticket.der at its Documents directory which is accessible via iTunes File Sharing.\n\nCompatibility:\n\niOS 11.0-12.1.2 on arm64/arm64e SoCs (A7-A12X)."
+"Refresh Icon Cache" = "เริ่มต้น Icon Cache ใหม่"
+"Description:\n\n" = "รายละเอียด:"
+"This option makes the jailbreak regenerate SpringBoard's system application installation cache to cause newly installed .app bundles to appear on the icon list." = "This option makes the jailbreak regenerate SpringBoard's system application installation cache to cause newly installed .app bundles to appear on the icon list."
+"iOS 11.0-12.2 and 12.4 on arm64/arm64e SoCs A7-A12X." = "iOS 11.0-12.2 and 12.4 on arm64/arm64e SoCs A7-A12X."
+"Disable Updates" = "ปิดการอัพเดทระบบ"
+"This option makes the jailbreak effectively disable the system's software update mechanism to prevent the system from automatically upgrading to the latest available firmware which may not be supported by the jailbreak at that time." = "This option makes the jailbreak effectively disable the system's software update mechanism to prevent the system from automatically upgrading to the latest available firmware which may not be supported by the jailbreak at that time."
```

Comment:
```suggestion
"This option makes the jailbreak effectively disable the system's software update mechanism to prevent the system from automatically upgrading to the latest available firmware which may not be supported by the jailbreak at that time." = "ตัวเลือกนี้ จะทำให้การเจลเบรคปิดกลไลการอัพเดตซอฟต์แวร์ของระบบ เพื่อป้องกันไม่ให้ระบบทำการอัพเกรดไปเป็นเฟิร์มแวร์ตัวล่าสุด ซึ่งอาจไม่รองรับการเจลเบรค ณ เวลานั้น"
```